frontend/src/components/VFeatureNotice/VFeatureNotice.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import VLink from "~/components/VLink.vue"
+
+defineProps<{
+  seeMoreHref: string
+}>()
+
+const emit = defineEmits<{
+  click: [MouseEvent]
+}>()
+
+const handleClick = (e: MouseEvent) => {
+  emit("click", e)
+}
+</script>
+
+<template>
+  <VLink
+    :href="seeMoreHref"
+    class="group flex h-10 w-full flex-row items-center gap-2 rounded-full border border-secondary pe-4 ps-2 hover:no-underline sm:w-[23.125rem]"
```

Comment:
I did this because it was "Fixed" in Figma. When Francisco uses "Fixed" over "Hug", I usually see that as a sign that text is meant to wrap in those scenarios.